src/common/password.h

```diff
@@ -58,11 +59,10 @@ namespace tools
     password_container& operator=(const password_container&) = delete;
     password_container& operator=(password_container&&) = default;
 
-    const std::string& password() const noexcept { return m_password; }
+    epee::wipeable_string password() const noexcept { return m_password; }
```

Comment:
This returns a copy now, instead of a reference. Is that intentional?

Reply:
It was, it was otherwise failing to perform some move, but it might be OK now that I added some move ctors around, I'll check whether it can be changed back.